src/css/layout/fixedDataTableCellLayout.css

```diff
@@ -50,6 +50,7 @@
   right: 0px;
   width: 6px;
   z-index: 1;
+  touch-action: none;
```

Comment:
same as above

also what's the pro of putting this on individual elements rather than the whole table?

Reply:
That's a good point. I was trying to isolate the warning and apply the rule only to the elements that caused it.
Some quick testing by adding the rule to the entire table does show that it removes the warning completely and it does not impact other functionalities.